lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3921,6 +3921,28 @@ def model_find_and_add_construction(model, climate_zone_set, intended_surface_ty
                                 'building_category' => building_category,
                                 'minimum_percent_of_surface' => minimum_percent_of_surface,
                                 'maximum_percent_of_surface' => maximum_percent_of_surface)
+
+      # 12/14/2021 added to test data pull from sub climate zones for transparent surfaces
```

Comment:
I think that this comment could be removed or at least rephrased to make the intent of the code below a bit more clear. The date can be removed.